src/components/20-molecules/datepicker/react/demo.jsx

```diff
@@ -0,0 +1,15 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DemoControlledDatepickerReact from './DemoDatepickerReact';
+import { withMarkdown } from '../../../../../.storybook/addons/markdown';
+import Readme from '../README.md';
+
+storiesOf('Molecules/Datepicker/React/Demos', module)
+  .addDecorator(withMarkdown(Readme))
+  .add('Controlled-component React-ified Datepicker with inputfield', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<DemoControlledDatepickerReact />, div);
```

Comment:
Just question, since this is handled differently anyways in our codebase:

We could make up the example in here instead of just calling the prepared demo-component. Since in our documentation, we show the user how to use skatejs and tell him to create a STATELESS react component and then use it everywhere and providing the state from the outside.

Meaning: The `DemoControlledDatepickerReact` component should only be what we suggest as skatejs wrapper, and the rest should be in here within the story, including the props, childfragments, etc

What do you think?

Reply:
It's not handled differently in our codebase, the approach was suggested by Marco, see his Button for more examples. It looks clean and simple.